grammars/css.cson

```diff
@@ -20,294 +20,1176 @@
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      # Timing-functions
+      {
+        'begin': '(?i)(?<![\\w-])(cubic-bezier|steps)(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'support.function.timing-function.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.timing-function.css'
+        'patterns': [
+          {
+            'match': '(?i)(?<![\\w-])(start|end)(?=\\s*\\)|$)'
+            'name': 'support.constant.step-direction.css'
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      # Transform functions
+      {
+        'begin': '''(?xi) (?<![\\w-])
+          ( (?:translate|scale|rotate)(?:[XYZ]|3D)?
+          | matrix(?:3D)?
+          | skew[XY]?
+          | perspective
+          )
+          (\\()
+        '''
+        'beginCaptures':
+          '1':
+            'name': 'support.function.transform.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'patterns': [
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+      {
+        'include': '#url'
+      }
+      # Variable expansion
+      {
+        'begin': '(?i)(?<![\\w-])(var)(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'support.function.misc.css'
+          '2':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'name': 'meta.function.variable.css'
+        'patterns': [
+          {
+            'name': 'variable.argument.css'
+            'match': '--[\\w-]+'
+          }
+          {
+            'include': '#property-values'
+          }
+        ]
+      }
+    ]
+  'functional-pseudo-classes':
+    'patterns': [
+      {
+        # Text direction
+        'begin': '(?i)((:)dir)(\\()'
+        'beginCaptures':
+          '1':
+            'name': 'entity.other.attribute-name.pseudo-class.css'
+          '2':
+            'name': 'punctuation.definition.entity.css'
+          '3':
+            'name': 'punctuation.section.function.begin.bracket.round.css'
+        'end': '\\)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.section.function.end.bracket.round.css'
+        'patterns': [
+          {
+            'match': '(?i)(?<![\\w-])(ltr|rtl)(?![\\w-])'
+            'name': 'support.constant.text-direction.css'
+          }
+          {
+            'include': '#property-value'
```

Comment:
No, but `#property-values` is also including comments, unquoted escape sequences, and other things that should be picked up anywhere.